CHANGELOG.md

```diff
@@ -38,6 +40,9 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
   textureSize()/imageSize() instead (`%s_dimensions` is still available for 2D
   textures)
 
+### Removed
+- `ngl_scene_freep` function; use `ngl_scene_unrefp` instead
+
```

Comment:
`api: add ngl_scene_unrefp() and use it` -> `api: replace ngl_scene_freep() with ngl_scene_unrefp()`

Reply:
fixed